boefjes/boefjes/katalogus/tests/test_plugins.py

```diff
@@ -110,12 +110,12 @@ def test_get_plugin(self):
         # Simpler endpoint works as well, but due to the mock the default mime_types are not dynamically added
         res = self.client.get("/v1/organisations/test-org/plugins/test-boefje-1")
         self.assertEqual(200, res.status_code)
-        assert "mime_types" in res.json()
-        assert not res.json()["mime_types"]
+        assert "produces" in res.json()
+        assert res.json()["produces"] == ["text/html"]
 
         # For boefjes that are pulled from the local repository, we actually get the default mime_types
-        assert set(self.client.get("/v1/organisations/test-org/plugins/kat_test").json()["mime_types"]) == set(
-            ["boefje/kat_test"]
+        assert set(self.client.get("/v1/organisations/test-org/plugins/kat_test").json()["produces"]) == set(
+            ["boefje/kat_test", "Hostname", "X509Certificate"]
```

Comment:
This should be a list without the Hostname and X509Certificate values right? As those are not mime-types

Reply:
Yeah this is because I didn't update the `boefje.json` for our mock/test boefje directory. Just updated it 👍